app/views/hosts/show.html.erb

```diff
@@ -46,9 +46,11 @@
           <%= render :partial => "hosts/metrics", :locals => { :report_summary => @report_summary[@host.name][:metrics] } %>
         <% end %>
       </div>
-      <div class="tab-pane" id="template" data-ajax-url='<%= templates_host_path(@host)%>'>
-        <%= spinner(_('Loading template information ...')) %>
-      </div>
+      <% if SETTINGS[:unattended] %>
```

Comment:
No, the behaviour does not change, look [here](https://github.com/theforeman/foreman/blob/develop/app/views/hosts/show.html.erb#L26). I just chose not to render content that cannot be seen. Or do we want a change to display 'templates' tab with a message when `unattended: false` ?